harness/determined/experimental/_native.py

```diff
@@ -287,30 +277,33 @@ def create(
     Arguments:
         trial_def:
             A class definition implementing the ``det.Trial`` interface.
+
         config:
             A dictionary representing the experiment configuration to be
             associated with the experiment.
-        mode:
-            The :py:class:`determined.experimental.Mode` used when creating
-            an experiment
 
-            1. ``Mode.CLUSTER`` (default): Submit the experiment to a remote
-            Determined cluster.
+        local:
+            A boolean indicating if training should happen locally. When
```

Comment:
1. "should" -> "will"
2. It would be good if you add one more sentence: When specified to be ``True``, the experiment will train on your local environment.